core/payment-driver/zksync/Cargo.toml

```diff
@@ -11,21 +11,21 @@ default = []
 async-trait = "0.1"
 anyhow = "1.0"
 awc = { version = "2.0", features = ["openssl"] }
-bigdecimal = { version = "0.1.0" }
+bigdecimal = { version = "0.2" }
 chrono = { version = "0.4", features = ["serde"] }
 futures3 = { version = "0.3", features = ["compat"], package = "futures" }
 hex = "0.4"
 lazy_static = "1.4"
 log = "0.4.8"
 maplit = "1.0"
-num = { version = "0.2", features = ["serde"] }
+num-bigint = { version = "0.3", features = ["serde"] }
 serde = "1.0"
 serde_json = "^1.0"
 tiny-keccak = "1.4.2"
 tokio = { version = "0.2", features = ["full"] }
 uuid = { version = "0.8", features = ["v4"] }
-zksync = { git = "https://github.com/matter-labs/zksync", rev = "9b4263c883c0f57c75068a0cc6339d051157a306"}
-zksync_eth_signer = { git = "https://github.com/matter-labs/zksync", rev = "9b4263c883c0f57c75068a0cc6339d051157a306"}
+zksync = { git = "https://github.com/matter-labs/zksync", branch = "master"}
+zksync_eth_signer = { git = "https://github.com/matter-labs/zksync", branch = "master"}
```

Comment:
Shouldn't we rather fix the version to a particular commit?

Reply:
yes, done :)